### DISCUSSION TITLE
6 to 7 migration question: release_name vs latest_release

### DISCUSSION THREAD

**Discussion by arderyp** [deployphp/deployer#3209] [Help needed]
The (very handy, thank you!) [migration documentation](https://deployer.org/docs/7.x/UPGRADE) says:

```
Find out next release name (ssh to the host, ls releases dir, find the biggest number). Example: 42.
Deploy with release_name:
dep deploy -o release_name=43

NOTE
In case there are multiple hosts with different release names, you should create a {{deploy_path}}/.dep/latest_release file in each host with the current release number of that particular host.
```

Is the second option an alternate to the first?  For example, if I set `{{deploy_path}}/.dep/latest_release` on all my hosts, can I issue `deploy` without `-o release_name=` on my first 7 deploy?

**Comment by antonmedv** [comment]
Yes, just set latest_release number to some arbitrary int so next time you deploy release_name is generate properly .

**Reply by arderyp** [reply]
Thanks!  However, it sounds from the documentation like I don't want to set it to some arbitrary int, but rather `LATEST-HOST-RELEASE-INT + 1`. Or any arbitrary int higher than that. Otherwise I may overwrite a pre-existing release, Right?

Also, if I do this, will deployer increment or delete `{{deploy_path}}/.dep/latest_release` for me after the deployment, assuming it's successful?

**Reply by antonmedv** [reply]
Right.

**Reply by arderyp** [reply]
Thanks.  Any thoughts on whether 7 is the recommended version currently, or not until RC is done?

**Reply by antonmedv** [reply]
7 is recommended.

**Reply by arderyp** [reply]
Thanks for the rapid responses and awesome package. Have a good weekend.
